### ISSUE TITLE
Add Up and Down Arrow reaction buttons to see more code

### ISSUE THREAD

**Issue by SimplyBallistic** [diogoscf/github-lines#4]

**Comment by diogoscf** [comment]
This is a really good idea, although I don't really know how this would work when a message has multiple snippets.

**Comment by SimplyBallistic** [comment]
> This is a really good idea, although I don't really know how this would work when a message has multiple snippets.

Maybe we could make the bot split them into multiple messages? Or just disable this feature for multi-snippets all together?

**Comment by diogoscf** [comment]
I certainly don't want to disable that feature, but splitting it up into multiple messages is a viable option

**Comment by SimplyBallistic** [comment]
> I certainly don't want to disable that feature, but splitting it up into multiple messages is a viable option

By disabling, I was referring to the up down arrows on multi snippets haha. What do you think is the best approach? 

1. Removing support for up down arrows on multi snippets
2. Splitting multi snippets to several messages

Once we work this out, I'll try make this feature once I get the time 🚀

**Comment by diogoscf** [comment]
Oh that makes more sense

I think splitting multi snippets into several messages is probably the best approach.

**Comment by Mondonno** [comment]
I agree! If someone wants to get more code to lookup (or the link line is linked to the line above).
Just i think to add the reaction collector, and the snippets will be deleted after a timeout!

Maybe my idea help :)
